requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy
-dask[array]==2022.2.1
+dask>=2022.2.1
 zmq
```

Comment:
this should be pyzmq. `zmq` just redirects to that

Reply:
see 
![image](https://user-images.githubusercontent.com/10111092/156420428-40d297b3-967f-4b3b-ada9-0766d353980d.png)

Reply:
lol yikes